espnet2/mt/frontend/embedding.py

```diff
@@ -6,6 +6,8 @@
 
 from typing import Tuple
 
+import joblib
+import numpy as np
```

Comment:
Are they used?

Reply:
Thanks for noticing it! I added by mistake.